genai_processors/contrib/__init__.py

```diff
@@ -12,4 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-"""Community contributed processors."""
+
+from .openrouter_model import OpenRouterModel
```

Comment:
Please don't import modules in __init__.py. This is contrib folder and we aim to add various integrations there. They will bring assorted dependencies with them. If we import them in __init__.py then all of them inevitably will be imported when accessing any of the contrib modules. At the very least this will slow down import and increas application footprint.

You don't need any changes in __init__.py. People can still import your module as `from genai_procesors.contrib import openrouter_model`.